### DISCUSSION TITLE
第6节课笔记(24班)

### DISCUSSION THREAD

**Discussion by zhjunqin** [InternLM/Tutorial#468] [笔记]
第6节课笔记(24班)

**Comment by kelenlv** [comment]
https://github.com/kelenlv/LLM_notes/blob/main/Day6.md

**Comment by waurblove** [comment]
https://www.bilibili.com/opus/888533136204365833?spm_id_from=333.999.0.0

**Comment by yezhou05** [comment]
https://app.yinxiang.com/fx/230c2c8a-f4d6-4879-971f-7455198f07c8

**Comment by yansuoyuli** [comment]
https://blog.csdn.net/m0_58431543/article/details/135760549

**Comment by ljn12yyds** [comment]
https://blog.csdn.net/ljn20001229/article/details/135774472?spm=1001.2014.3001.5501

**Comment by flh1107** [comment]
https://www.bilibili.com/read/cv29982912/

**Comment by lindsey-chang** [comment]
https://blog.csdn.net/qq_37397652/article/details/135812669

**Comment by ch521-od** [comment]
https://blog.csdn.net/qq_74261455/article/details/135845517?spm=1001.2014.3001.5501

**Comment by robotzm** [comment]
https://blog.csdn.net/weixin_48319544/article/details/135843323?spm=1001.2014.3001.5501

**Comment by Xinyi-Cheng** [comment]
https://zhuanlan.zhihu.com/p/679781266
